tools/bin/commands/system-test

```diff
@@ -312,7 +312,7 @@ test_build() {
     export NAMESPACE_USE_EXISTING=$project
     export KUBERNETES_NAMESPACE=$project
     export OPENSHIFT_TEMPLATE_FROM_WORKSPACE=true
-    export WORKSPACE=deploy
+    export WORKSPACE=$(basedir)/../..
```

Comment:
If you are looking for `syndesis/app` then you can easily use $(appdir) which even yields an absolute path without ../..

Reply:
perfect, will update the pr.